ironfish-cli/src/utils/s3.ts

```diff
@@ -309,6 +318,31 @@ export function getS3Client(
   })
 }
 
+export function getR2S3Client(credentials: {
+  r2AccessKeyId: string
+  r2SecretAccessKey: string
+}): S3Client {
+  return new S3Client({
+    region: 'auto',
+    endpoint: R2_ENDPOINT,
+    credentials: {
+      accessKeyId: credentials.r2AccessKeyId,
+      secretAccessKey: credentials.r2SecretAccessKey,
+    },
+  })
+}
```

Comment:
These are binary compatible with S3, so I wouldnt expect this to exist. I would just expect these to be parameters that you pass into getS3Client but we can refactor this later.